crates/bevy_ecs/src/event.rs

```diff
@@ -264,6 +274,10 @@ impl<'w, 's, E: Event> EventReader<'w, 's, E> {
 /// # bevy_ecs::system::assert_is_system(my_system);
 /// ```
 ///
+/// Note that a system will panic if an `EventWriter` is used as a `SystemParam` without a
+/// corresponding [`Events<E>`] resource in the [`World`]. For cases where the `Events` resource
```

Comment:
Try to align your line breaks with the end of sentence / clause; it make it much easier to diff the docs later.